config/qt5_core.mpb

```diff
@@ -0,0 +1,56 @@
+project {
+  expand(QT5_INCDIR) {
+    $QT5_INCDIR
+    $(QT5_DIR)/include
+  }
+
+  requires += qt5
+  includes += $(QT5_INCDIR)
+  libpaths += $(QT5_DIR)/lib $(QT5_DIR)/bin
+  lit_libs += Qt5Core
+
+  verbatim (make, macros) {
+    CPPFLAGS += -fPIC
```

Comment:
Add a comment with link to Qt's documentation for why this is here.  We should really have something that's not so compiler-specific.

Reply:
I can't find definitive Qt documentation that says this other than a [reference to the fact that it's default in Qt5 in an official CMake guide](http://doc.qt.io/archives/qt-5.5/cmake-manual.html) and [the cpp code that stops the compiler if building without -fPIC](http://code.qt.io/cgit/qt/qtbase.git/tree/src/corelib/global/qglobal.h#n1177). There are a lot of forum and github issues related to this though.

In regards to it being gcc specific, is there anything I can tell MPC that would be equivalent to [CMake's POSITION_INDEPENDENT_CODE](https://cmake.org/cmake/help/latest/prop_tgt/POSITION_INDEPENDENT_CODE.html)?